exec/cnex/global.c

```diff
@@ -114,9 +123,40 @@ static FILE *check_file(TExecutor *exec, void *pf)
     return f;
 }
 
-void global_init()
+void global_init(int argc, char *argv[], int iArgStart)
 {
     random_initModule();
+    VAR_args.address = cell_createArrayCell(0);
+    sys_initModule(argc, argv, iArgStart, VAR_args.address);
+    BuiltinVariables[0].value = VAR_args.address;
+}
+
+void global_shutdown()
+{
+    // ToDo: Figure out which method will actually free the global variables
```

Comment:
Has this been figured out? If this commented code is not required, remove it.

Reply:
Not entirely.  Using the call below to `cell_clearCell()` also `free()`'s the Array in the Cell; but I still end up leaking memory on shutdown, which isn't a huge deal, but it is distracting.  I think I'm going to attempt a Valgind on this code to see if it can tell me more about what is actually happening here.